File: MODUL-6/SOAL 5/PRAK605-2410817120017-RIKAFAULIANARAHMI.py
n = int(input())

print("Matriks A")
A = [list(map(int, input().split())) for _ in range(n)]

print("Matriks B")
B = [list(map(int, input().split())) for _ in range(n)]

hasil = [[sum(A[i][k] * B[k][j] for k in range(n)) for j in range(n)] for i in range(n)]

print("Matriks AXB")
for baris in hasil:
    print(" ".join(map(str, baris)))

File: MODUL-6/SOAL 5/PRAK605-2410817120017-RIKAFAULIANARAHMI.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);

    int A[n][n], B[n][n], hasil[n][n];

    printf("Matriks A\n");
    for (int i = 0; i < n; i++)
        for (int j = 0; j < n; j++)
            scanf("%d", &A[i][j]);

    printf("Matriks B\n");
    for (int i = 0; i < n; i++)
        for (int j = 0; j < n; j++)
            scanf("%d", &B[i][j]);

    for (int i = 0; i < n; i++)
        for (int j = 0; j < n; j++) {
            hasil[i][j] = 0;
            for (int k = 0; k < n; k++)
                hasil[i][j] += A[i][k] * B[k][j];
        }

    printf("Matriks AXB\n");
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++)
            printf("%d ", hasil[i][j]);
        printf("\n");
    }

    return 0;
}
